enums/Cargo.toml

```diff
@@ -10,9 +10,11 @@ cc = "^1.0"
 phf_codegen = "^0.8"
 
 [dependencies]
-tree-sitter = "^0.16"
```

Comment:
Could you update the tree-sitter dependency in a separate PR (which I will land first)?

Reply:
Done in #451